corehq/ex-submodules/fluff/pillow.py

```diff
@@ -137,3 +133,21 @@ def _get_indicator_doc_from_class_and_id(indicator_class, doc_id):
         return indicator_class.get(indicator_id)
     except ResourceNotFound:
         return indicator_class(_id=indicator_id)
+
+
+def get_fluff_pillow(indicator_class, delete_filtered=False, chunk_size=None):
+    processor = FluffPillowProcessor(indicator_class, delete_filtered=delete_filtered, chunk_size=chunk_size)
+
+    name = '{}Pillow'.format(indicator_class.__name__)
+    checkpoint = PillowCheckpoint('fluff.{}.{}'.format(name, get_machine_id()))
```

Comment:
i think this is fine. the other approach if we wanted to get rid of the legacy names would be to do a migration [like this](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cleanup/migrations/0010_rename_default_change_feed_checkpoint.py#L8)